server/protocols/jmap/src/test/java/org/apache/james/jmap/FirstUserConnectionFilterThreadTest.java

```diff
@@ -28,15 +28,11 @@
 import org.apache.james.mailbox.MessageManager;
 import org.apache.james.mailbox.exception.BadCredentialsException;
 import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.mailbox.model.*;
```

Comment:
Wildcard. You can set this up in your Intellij settings (java -> import I believe)